src/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.8"
+
+services:
+
+  splunk:
+    image: ${SPLUNK_IMAGE:-splunk/splunk:latest}
+    container_name: splunk
+    environment:
+      - SPLUNK_START_ARGS=--accept-license
+      - SPLUNK_HEC_TOKEN=traffic
+      - SPLUNK_PASSWORD
+    ports:
+      - 8000:8000
+      - 8088:8088

```

Comment:
I think the docker-compose file should be in the root and not root of src folder because  we can use it to launch other services that may have config / files outside src.